src/common/UnitTests-CommonLib/UnitTestsVersionHelper.cpp

```diff
@@ -10,18 +10,18 @@ namespace UnitTestsVersionHelper
     const int MINOR_VERSION_12 = 12;
     const int REVISION_VERSION_0 = 0;
 
-    TEST_CLASS(UnitTestsVersionHelper)
+    TEST_CLASS (UnitTestsVersionHelper)
```

Comment:
@SeraphimaZ 
FYI: the auto-formatting has introduced a small change in the test files format